README.md

```diff
@@ -218,40 +218,64 @@ Configuration of Tensorflow for a model is done through the Parameters section o
 * `TF_GRAPH_TAG`: Tag of the graphs to use. See [protobuf here](https://github.com/tensorflow/tensorflow/blob/6f72753a66d6abab8b839cc263a9f1329861f6f9/tensorflow/core/protobuf/meta_graph.proto#L56)
 * `TF_SIGNATURE_DEF`: Signature def to use. See [protobuf here](https://github.com/tensorflow/tensorflow/blob/6f72753a66d6abab8b839cc263a9f1329861f6f9/tensorflow/core/protobuf/meta_graph.proto#L260-L331)
 * `MAX_SESSION_SHARE_COUNT`: This parameter specifies the maximum number of [model instances](https://github.com/triton-inference-server/server/blob/main/docs/model_configuration.md#instance-groups) that can share a [TF session](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/core/public/session.h). The default value is 1 which means Triton will create a separate TF session for each model instance. If this parameter is set to the total number of instances, then Triton will create only a single TF session which will be shared by all the instances. Sharing TF sessions among model instances can reduce memory footprint of loading and executing the model.
+* `TF_INIT_OPS_FILE`: This parameter specifies the name of the file in JSON
```

Comment:
Should update the documentation to match the current behavior. 
Say that json file can be provided per version directory or model directory. Talk about which one will take precedence and then give some examples.